assets/src/components/map/follower.tsx

```diff
@@ -234,6 +223,9 @@ export const drawerOffsetAutoCenter =
     }
   }
 
-export const fixedZoomDrawerOffsetAutoCenter = drawerOffsetAutoCenter(false)
+export const fixedZoomDrawerOffsetAutoCenter = drawerOffsetAutoCenter(
+  false,
+  [0, 0]
+)
```

Comment:
related to snapshot changes
```suggestion
    // ```
    // vpcElement.getBoundingClientRect().right - mapElement.getBoundingClientRect().left
    //  -> 445
    // ```
    // Create a new inner bounds from the map bounds + "padding" to shrink the
    // inner bounds
    // In this case, we get the top left of the inner bounds by padding the left
    // with the distance from the right side of the VPC to the left side of the
    // map container
    new Point(445, 0)
)
```

Reply:
Oooh I did change the defaults, didn't I

Reply:
Related question - is this actually a problem? Having a non-zero default creates cognitive overhead, and maybe we don't need it?

Reply:
Yes & no. If we don't need this default, we don't actually need `fixedZoomDrawerOffsetAutoCenter` at all, but then we need (and probably should) update all usages of `fixedZoomDrawerOffsetAutoCenter` to actually use `drawerOffsetAutoCenter`

or `fixedZoomDrawerOffsetAutoCenter` should forward the `topLeft` parameter, since it is setting `useCurrentZoom` for a reason.

Reply:
Yeah, I just tested out location search, and that's still broken, so I think you're right that we need to rethink `fixedZoomDrawerOffsetAutoCenter`.

Reply:
(Mentioned in Slack, and mentioning here for posterity as well...) the change suggested here doesn't affect the snapshot diff.